cypress.json

```diff
@@ -12,7 +12,7 @@
   "viewportWidth": 2560,
   "viewportHeight": 1440,
   "env": {
-    "localTestURL": "http://go.test",
```

Comment:
Wouldn't it be http://localhost:8889 ?

Reply:
This URL is what WordPress core is installed with, and the apache config is updated to show the site at http://go.test

https://github.com/godaddy-wordpress/go/blob/ddf2e9562ed89a2932efba57469078900af39331/.dev/deploy-scripts/install-wp-tests.sh#L115

https://github.com/godaddy-wordpress/go/blob/ddf2e9562ed89a2932efba57469078900af39331/.dev/tests/apache-ci.conf#L4